toolbox/py_xnvme_pkg/Makefile

```diff
@@ -0,0 +1,123 @@
+#
+# This Makefile serves as convenient command-line auto-completion when working on xNVMe-ctypes
+#
+#
+TOOLBOX_DIR?=../
+PROJECT_NAME=xnvme
+XNVME_CAPI_HEADERS=$(shell ls ../../include/libxnvme* | grep -v util | grep -v libxnvmec)
+XNVME_SO_PATH=$(shell pkg-config xnvme --variable=libdir)/libxnvme-shared.so
```

Comment:
This is Linux specific. I would also recommend, that we look for the lib in the local repo. For example on Mac: `$GIT_ROOT/builddir/lib/libxnvme-shared.dylib`

Reply:
Good spot. Usage of this is completely removed, it now relies on the library being installed on the system as that is where ``clang2py`` looked for headers anyway, so the library and the headers are needed there regardless.